core/model/modx/moduser.class.php

```diff
@@ -81,6 +85,7 @@ public function setSudo($sudo) {
      */
     public function save($cacheFlag = false) {
         $isNew = $this->isNew();
+        if ($isNew) $this->set('createdon', time());
```

Comment:
When dealing with import scripts and such, checks like this prevent you from setting a predefined createdon date. It might be better if this is changed to something like:

```` php
if ($isNew && $this->get('createdon') < 1) {
   $this->set('createdon', time());
}
````

That way if the createdon is already set, it isn't overwritten.